spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -1223,7 +1223,7 @@ public JSONObject listLibrariesDataTable(HttpSession session, JSONObject json) {
         String identificationBarcode = library.getIdentificationBarcode();
         
         jsonArray.add(JsonSanitizer.sanitize("[\"" + library.getName() + "\",\"" + library.getAlias() + "\",\"" + library.getLibraryType().getDescription() + "\",\""
-            + library.getSample().getName() + "\",\"" + qcpassed + "\",\"" + "<a href=\"/miso/library/" + library.getId()
+            + library.getSample().getName() + " (" + library.getSample().getAlias() + ")" + "\",\"" + qcpassed + "\",\"" + "<a href=\"/miso/library/" + library.getId()
             + "\"><span class=\"ui-icon ui-icon-pencil\"></span></a>" + "\",\""
```

Comment:
The "Fix broken things" pull request covers the conversion to JSONArray instead of escaping everything.